pants

```diff
@@ -205,11 +209,8 @@ pants_dir="$(bootstrap_pants "${pants_version}" "${python}")"
 # See https://blog.phusion.nl/2017/10/13/why-ruby-app-servers-break-on-macos-high-sierra-and-what-can-be-done-about-it/
 export no_proxy='*'
 
-if [[ -z "${pants_config_file}" ]]; then
-  pants_config_file_flag="";
-else
-  pants_config_file_flag=--pants-config-files="[\"${PWD}/${pants_config_file}\"]";
+if ! [[ -v PANTS_CONFIG_FILES ]] && [[ -n ${pants_config_file} ]]; then
```

Comment:
If someone sets `PANTS_CONFIG_FILES` to an append value, this won't do the right thing... but I don't see a great way to do this without parsing the config file. Maybe not the end of the world.

Is it worth adding something native to handle this case? I don't know exactly what it would look like...

Reply:
I mean, TBH we can also just not do this at all, and say "If you set PANTS_TOML or PANTS_INI that only tells the runner script where to look for the pants version, you still have to tell pants itself to use that file some other way". Maybe that's better/simpler? If you're manipulating PANTS_TOML/PANTS_INI you're already doing something advanced.

Reply:
This was supposed to be a convenience, but it's turning into an inconvenience...

Reply:
Yeah, I think I've convinced myself.

Reply:
Yeah, I agree with you.

Reply:
OK, simplified this. PTAL.